code/modules/mob/living/carbon/human/human_examine.dm

```diff
@@ -1,170 +1,70 @@
-		if(400 SECONDS to 600 SECONDS)
-			msg += "<span class='warning'>[p_they(TRUE)] [p_are()] extremely jittery.</span>\n"
-		if(200 SECONDS to 400 SECONDS)
-			msg += "<span class='warning'>[p_they(TRUE)] [p_are()] twitching ever so slightly.</span>\n"
+/mob/living/carbon/human/examine_handle_individual_limb(limb_name)
+	var/msg = ""
+	switch(limb_name)
+		if("hands")
+			if(blood_DNA)
+				return "<span class='warning'>[p_they(TRUE)] [p_have()] [hand_blood_color != "#030303" ? "blood-stained":"oil-stained"] hands!</span>\n"
+		if("eyes")
+			if(HAS_TRAIT(src, SCRYING))
+				if(iscultist(src) && HAS_TRAIT(src, CULT_EYES))
+					return "<span class='boldwarning'>[p_their(TRUE)] glowing red eyes are glazed over!</span>\n"
+				return "<span class='boldwarning'>[p_their(TRUE)] eyes are glazed over.</span>\n"
+			else
+				if(iscultist(src) && HAS_TRAIT(src, CULT_EYES))
+					return "<span class='boldwarning'>[p_their(TRUE)] eyes are glowing an unnatural red!</span>\n"
 
+	return msg
 
-	var/appears_dead = FALSE
-	var/just_sleeping = FALSE //We don't appear as dead upon casual examination, just sleeping
+/mob/living/carbon/human/examine_what_am_i(skip_gloves = FALSE, skip_suit_storage = FALSE, skip_jumpsuit = FALSE, skip_shoes = FALSE, skip_mask = FALSE, skip_ears = FALSE, skip_eyes = FALSE, skip_face = FALSE)
+	if(!dna)
+		return
 
-	if(stat == DEAD || HAS_TRAIT(src, TRAIT_FAKEDEATH))
-		var/obj/item/clothing/glasses/E = get_item_by_slot(slot_glasses)
-		var/are_we_in_weekend_at_bernies = E?.tint && istype(buckled, /obj/structure/chair/wheelchair) //Are we in a wheelchair with our eyes obscured?
+	var/msg = ""
 
-		if(isliving(user) && are_we_in_weekend_at_bernies)
-			just_sleeping = TRUE
-		else
-			appears_dead = TRUE
-
-		if(suiciding)
-			msg += "<span class='warning'>[p_they(TRUE)] appear[p_s()] to have committed suicide... there is no hope of recovery.</span>\n"
-		if(!just_sleeping)
-			msg += "<span class='deadsay'>[p_they(TRUE)] [p_are()] limp and unresponsive; there are no signs of life"
-			if(get_int_organ(/obj/item/organ/internal/brain))
-				if(!key)
-					var/foundghost = FALSE
-					if(mind)
-						for(var/mob/dead/observer/G in GLOB.player_list)
-							if(G.mind == mind)
-								foundghost = TRUE
-								if(G.can_reenter_corpse == 0)
-									foundghost = FALSE
-								break
-					if(!foundghost)
-						msg += " and [p_their()] soul has departed"
-			msg += "...</span>\n"
-	if(!get_int_organ(/obj/item/organ/internal/brain))
-		msg += "<span class='deadsay'>It appears that [p_their()] brain is missing...</span>\n"
-
-	msg += "<span class='warning'>"
+	var/displayed_species = dna?.species.name
+	var/examine_color = dna.species.flesh_color
+	for(var/obj/item/clothing/C in src)			//Disguise checks
+		if(C == head || C == wear_suit || C == wear_mask || C == w_uniform || C == belt || C == back)
+			if(C.species_disguise)
+				displayed_species = C.species_disguise
```

Comment:
```suggestion
				displayed_species = C.species_disguise
				break
```
Prob want a break after this. Or some other handling of multiple clothing with `species_disguise` set